.github/workflows/linux.yml

```diff
@@ -204,13 +200,15 @@ jobs:
       OV_TOKENIZERS_INSTALL_DIR: ${{ github.workspace }}/openvino_tokenizers/install
       OPENVINO_TOKENIZERS_REPO: ${{ github.workspace }}/openvino_tokenizers
       BUILD_DIR: ${{ github.workspace }}/openvino_tokenizers/build
+      OV_ROOT: ${{ github.workspace }}/openvino/install/${{ needs.openvino_download.outputs.ov_package_path }}
```

Comment:
we already use INSTALL_DIR, can we do not introduce one more variable?

Reply:
Yes, but ${{ needs.openvino_download.outputs.ov_package_path }} will still have to be included everywhere where we need to point to the folder with setupvars, so I've added a separate variable just to make commands less long. 

Why ov_package_path is returned is explained in this discussion: https://github.com/openvinotoolkit/openvino/pull/26050#discussion_r1716659110